airframe-config/src/main/scala/wvlet/config/YamlReader.scala

```diff
@@ -72,14 +72,7 @@ object YamlReader extends LogSupport {
     val builder = ObjectBuilder(surface)
     if (prop != null) {
       for ((k, v) <- prop.asScala) {
-        v match {
-          case al: java.util.ArrayList[_] =>
-            for (a <- al.asScala) {
-              builder.set(k.toString, a)
-            }
-          case _ =>
-            builder.set(k.toString, v)
-        }
+        builder.set(k.toString, v)
```

Comment:
Simplified the usage of object builder by supporting adding single element or collection.